### PR TITLE
Hide core contributors section if there's no core contributors

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4252

**Screen-cast**

Peek 2021-05-10 08-55

### CODE REVIEW

**Review by znarf — APPROVED** [review]
Looks good to me. 

@Betree do we have a better practice to ignore empty section?

File: lib/collective-sections.js [hunk 1]
```diff
@@ -217,6 +217,9 @@ const getSectionsToRemoveForUser = (collective, isAdmin) => {
   if (isEmptyCollectiveLocation(collective)) {
     toRemove.add(Sections.LOCATION);
   }
+  if (collective.coreContributors?.length === 0) {
```
Comment (znarf) [inline lib/collective-sections.js:220]: That's not doing what we want, It should be visible if there are financial contributors for example.

**Review by znarf — CHANGES_REQUESTED** [review]
It should be displayed if financial contributors.

File: lib/collective-sections.js (hunk 1 above)
Comment (SudharakaP) [reply lib/collective-sections.js]: @znarf : Ops, I didn't think about the financial contributors. I've modified the check. Thanks for pointing it out. 😄

**Pull request merged**